tmt/hardware.py

```diff
@@ -737,7 +737,7 @@ def _cast_int(raw_value: Any) -> int:
             )
 
 
-class NumberConstraint(Constraint[float]):
+class NumberConstraint(Constraint['Size']):
```

Comment:
hmm, I might be missing something. Why this change?

Reply:
Doesn't look correct indeed.

Reply:
Same here ;)

Reply:
update:https://github.com/teemtee/tmt/pull/3297/commits/7df0e709e1addbd4ebc62955511e43de37fd341a.

Sorry,mentor, I somehow overlooked the later part of the sentence :sweat_smile: